docs/src/guides/hub-entities.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Hub Entities
+navTitle: Hub Entities
+description: Core Entities in ArcGIS Hub
+order: 3
+group: 2-concepts
+---
+
+# Hub Entities
+
+While ArcGIS Hub acts as an integration point for the entire ArcGIS ecosystem, at it's core are 5 main entities:
+
+- Hub Sites
+- Hub Pages
+- Hub Initiatives
+- Hub Projects
+- Hub Events
+
+All other ArcGIS content is handled generically as "Hub Content"
+
+While Discussions are a key component of the ArcGIS Hub as a system, they are in independent subsystem, documented in the [Hub Discussions Guide](/hub.js/guides/hub-discussions)
+
+The following sections describe these entites, their role in the Hub System, and how you work with them in Hub.js
+
+### Hub Sites
+
+In terms of ArcGIS Hub, sites are the central entity that our customers work with. A site is backed by an item, it has a [layout](/hub.js/api/common/IHubLayout/) that can include a header and footer, and a theme.
```

Comment:
Change "our customers" to an external meaningful phrase. I doubt GitHub says "our customers" in their dev docs.